examples/reliable_conversation/src/tasks/task_registry.py

```
@@ -0,0 +1,19 @@
+"""
+Task registry for RCM quality control tasks.
+Registers all tasks with the app instance.
+"""
+
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)